core/modules/main/templates/_issuelogitem.inc.php

```diff
@@ -48,6 +48,7 @@
                             $previous_value = ($item->getPreviousValue()) ? (($old_item = \thebuggenie\core\entities\Status::getB2DBTable()->selectById($item->getPreviousValue())) ? __($old_item->getName()) : __('Unknown')) : __('Not determined');
                             $new_value = ($item->getCurrentValue()) ? (($new_item = \thebuggenie\core\entities\Status::getB2DBTable()->selectById($item->getCurrentValue())) ? __($new_item->getName()) : __('Unknown')) : __('Not determined');
                             echo __("Status changed: %previous_value => %new_value", array('%previous_value' => '<strong>'.$previous_value.'</strong>', '%new_value' => '<strong>'.$new_value.'</strong>'));
+	                        echo __(" by  %who_value", array('%who_value' => '<strong>'.$item->getUser().'</strong>'));
```

Comment:
This should be combined with the text in the original string. "By *name*" doesn't make sense as a separate sentence on its own. Feel free to alter existing strings if that's required.